CHANGELOG.md

```diff
@@ -2,10 +2,349 @@
+  [#3676](https://github.com/mixxxdj/mixxx/pull/3676)
+  [#3880](https://github.com/mixxxdj/mixxx/pull/3880)
+  [#4085](https://github.com/mixxxdj/mixxx/pull/4085)
+
+
+### Skins ###
+* Add experimental QML skin
+  [#3345](https://github.com/mixxxdj/mixxx/pull/3345)
+  [#3446](https://github.com/mixxxdj/mixxx/pull/3446)
+  [#3854](https://github.com/mixxxdj/mixxx/pull/3854)
+  [#3891](https://github.com/mixxxdj/mixxx/pull/3891)
+  [#3915](https://github.com/mixxxdj/mixxx/pull/3915)
+  [#3894](https://github.com/mixxxdj/mixxx/pull/3894)
+  [#3920](https://github.com/mixxxdj/mixxx/pull/3920)
+  [#3907](https://github.com/mixxxdj/mixxx/pull/3907)
+  [#3925](https://github.com/mixxxdj/mixxx/pull/3925)
+  [#3928](https://github.com/mixxxdj/mixxx/pull/3928)
+  [#3932](https://github.com/mixxxdj/mixxx/pull/3932)
+  [#3911](https://github.com/mixxxdj/mixxx/pull/3911)
+  [#3937](https://github.com/mixxxdj/mixxx/pull/3937)
+  [#3940](https://github.com/mixxxdj/mixxx/pull/3940)
+  [#3913](https://github.com/mixxxdj/mixxx/pull/3913)
+  [#3950](https://github.com/mixxxdj/mixxx/pull/3950)
+  [#3919](https://github.com/mixxxdj/mixxx/pull/3919)
+  [#3955](https://github.com/mixxxdj/mixxx/pull/3955)
+  [#3957](https://github.com/mixxxdj/mixxx/pull/3957)
+  [#3961](https://github.com/mixxxdj/mixxx/pull/3961)
+  [#3952](https://github.com/mixxxdj/mixxx/pull/3952)
+  [#3963](https://github.com/mixxxdj/mixxx/pull/3963)
+  [#3971](https://github.com/mixxxdj/mixxx/pull/3971)
+  [#3959](https://github.com/mixxxdj/mixxx/pull/3959)
+  [#3972](https://github.com/mixxxdj/mixxx/pull/3972)
+  [#3992](https://github.com/mixxxdj/mixxx/pull/3992)
+  [#4003](https://github.com/mixxxdj/mixxx/pull/4003)
+  [#4004](https://github.com/mixxxdj/mixxx/pull/4004)
+  [#3999](https://github.com/mixxxdj/mixxx/pull/3999)
+  [#4000](https://github.com/mixxxdj/mixxx/pull/4000)
+  [#4067](https://github.com/mixxxdj/mixxx/pull/4067)
+  [#4068](https://github.com/mixxxdj/mixxx/pull/4068)
+  [#4060](https://github.com/mixxxdj/mixxx/pull/4060)
+  [#4037](https://github.com/mixxxdj/mixxx/pull/4037)
+  [#3934](https://github.com/mixxxdj/mixxx/pull/3934)
+  [#4117](https://github.com/mixxxdj/mixxx/pull/4117)
+* Add new "RGB Stacked" waveform [#3153](https://github.com/mixxxdj/mixxx/pull/3153)
+* Add harmonic keywheel window
+  [#1695](https://github.com/mixxxdj/mixxx/pull/1695)
+  [#3622](https://github.com/mixxxdj/mixxx/pull/3622)
+  [#3624](https://github.com/mixxxdj/mixxx/pull/3624)
+* Make beat indicator control behaviour more natural [#3608](https://github.com/mixxxdj/mixxx/pull/3608)
+* Fix crash if no skin is available
+  [#3918](https://github.com/mixxxdj/mixxx/pull/3918)
+  [#3939](https://github.com/mixxxdj/mixxx/pull/3939)
+* Don't allow waveforms to obtain input focus [#4128](https://github.com/mixxxdj/mixxx/pull/4128)
+* Inverted scroll wheel waveform zoom direction to mach other applications [#4195](https://github.com/mixxxdj/mixxx/pull/4195)
```

Comment:
already part of 2.3.1

Reply:
No, it was merged to main.

Reply:
My bad, #4195 did only go into main, but #4128 was opened against 2.3 as well (#4134).

Reply:
yep, the fix was back-ported to 2.3